.github/workflows/kind.yml

```diff
@@ -68,9 +68,7 @@ jobs:
           kubectl wait --for=condition=Complete --timeout=60s job/sample-job
       - name: copy telemetry trace output
         run: |
-          kubectl cp -c filecp default/test-opentelemetry-collector-0:tmp/trace.json ./test/e2e/${{ matrix.library }}/traces.json.tmp
-          jq 'del(.resourceSpans[].scopeSpans[].spans[].endTimeUnixNano, .resourceSpans[].scopeSpans[].spans[].startTimeUnixNano) | .resourceSpans[].scopeSpans[].spans[].spanId|= (if . != "" then "xxxxx" else . end) | .resourceSpans[].scopeSpans[].spans[].traceId|= (if . != "" then "xxxxx" else . end) | .resourceSpans[].scopeSpans|=sort_by(.scope.name)' ./test/e2e/${{ matrix.library }}/traces.json.tmp | jq --sort-keys . > ./test/e2e/${{ matrix.library }}/traces.json
```

Comment:
I am +1 any solution that looks better than my jq monstrosity here hah

Reply:
In fairness to your jq monstrosity, @damemi, this experiment breaks things up into a bunch of small jq monsters that form up into bigger jq monsters with bash and bats glue. It's like a different flavor of poison: we get to choose the side-effects.

Reply:
Today I Learned that `jq` has no cares about arbitrary whitespace, so [the monster is BACK](https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/121/files#diff-93a3e09daea561b2dfc5c4c85465654bbf4470a640c139a97ae0bc638586f625R40-R55) but after a bath, some hair combing, and some ribbons put on!